docs: Add README to Solve Too Many Openfiles in MacOS (#23207)
kubernetes/website: README.md

This will start the local Hugo server on port 1313. Open up your browser to http://localhost:1313 to view the website. As you make changes to the source files, Hugo updates the website and forces a browser refresh.

### Troubleshooting in MacOS for Openfile Problems

If you find error looks like the following error:

```
Built in 16332 ms
Watching for changes in /path/to/website/{archetypes,assets,content,data,i18n,layouts,static,themes}
Watching for config changes in /path/to/website/config.toml, /path/to/website/themes/docsy/config.toml, /path/to/website/go.mod
Environment: "development"
Serving pages from memory
Running in Fast Render Mode. For full rebuilds on change: hugo server --disableFastRender
Web Server is available at http://localhost:1313/ (bind address 127.0.0.1)
Press Ctrl+C to stop
ERROR 2020/08/01 19:09:18 Error: listen tcp 127.0.0.1:1313: socket: too many open files
make: *** [serve] Error 1
```

> We can try to check openfiles current limit:

Reviewer: Avoid ["we"](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-using-we):
```suggestion
Try checking the current limit for open files:
```

`launchctl limit maxfiles`

> And then we can do this:

Reviewer: ```suggestion
Then run the following commands:
```

```
#!/bin/sh

# These are the original gist links, linking to my gists now.
# curl -O https://gist.githubusercontent.com/a2ikm/761c2ab02b7b3935679e55af5d81786a/raw/ab644cb92f216c019a2f032bbf25e258b01d87f9/limit.maxfiles.plist
# curl -O https://gist.githubusercontent.com/a2ikm/761c2ab02b7b3935679e55af5d81786a/raw/ab644cb92f216c019a2f032bbf25e258b01d87f9/limit.maxproc.plist

curl -O https://gist.githubusercontent.com/tombigel/d503800a282fcadbee14b537735d202c/raw/ed73cacf82906fdde59976a0c8248cce8b44f906/limit.maxfiles.plist
curl -O https://gist.githubusercontent.com/tombigel/d503800a282fcadbee14b537735d202c/raw/ed73cacf82906fdde59976a0c8248cce8b44f906/limit.maxproc.plist

sudo mv limit.maxfiles.plist /Library/LaunchDaemons
sudo mv limit.maxproc.plist /Library/LaunchDaemons

sudo chown root:wheel /Library/LaunchDaemons/limit.maxfiles.plist
sudo chown root:wheel /Library/LaunchDaemons/limit.maxproc.plist

sudo launchctl load -w /Library/LaunchDaemons/limit.maxfiles.plist
```

> This is working for Catalina as well as Mojave MacOS.

Reviewer: ```suggestion
This works for Catalina as well as Mojave MacOS.
```
